src/config.ts

```diff
@@ -437,12 +437,22 @@ async function composeSingleInputConfig<T extends CliFlags>(
 ): Promise<MergedConfig> {
   debug('entering single entry config mode');
 
-  const sourcePath = path.resolve(cliFlags.input);
-  const workspaceDir = path.dirname(sourcePath);
+  let sourcePath: string;
+  let workspaceDir: string;
+  let input: InputFormat;
   const entries: ParsedEntry[] = [];
   const exportAliases: { source: string; target: string }[] = [];
   const tmpPrefix = `.vs-${Date.now()}.`;
-  const input = detectInputFormat(sourcePath);
+
+  if (cliFlags.input && /https?:\/\//.test(cliFlags.input)) {
+    sourcePath = cliFlags.input;
+    input = { format: 'html', entry: sourcePath };
+    workspaceDir = '.';
```

Comment:
`workspaceDir` はこの時点で絶対パスになっている必要があります。この場合どのパスを指定すべきかわかりませんが…ひとまず `process.cwd()` などとしてください。
```suggestion
    workspaceDir = process.cwd();
```

Reply:
composeProjectConfig関数に合わせて

```
workspaceDir = otherConfig.workspaceDir;
```

という実装もできるかと思ったのですが、結果が以下のようになってしまったので、とりあえずprocess.cwd()にしておきます。
```
 preview/Users/takaiayumu/IdeaProjects/vivliostyle-cli/https:/vivliostyle.github.io/vivliostyle_doc/ja
```